src/amber/cli/commands/new.cr

```diff
@@ -22,6 +22,12 @@ module Amber::CLI
 
       def run
         Amber::CLI.color = !options.no_color?
+        full_path_name = File.join(`pwd`.sub("\n", ""), args.name)
+        if full_path_name.includes?(" ")
```

Comment:
Maybe you could use here a regex, just imagine the user inserts a wonderful tab character.

Using a regex:

`full_path_name =~ /s/`

Reply:
Can we change command `pwd` by some other native crystal method?

Something like `Dir.current`

Reply:
Not sure tabs can even be in a path. Never seen that before...

Dir.current sounds good.